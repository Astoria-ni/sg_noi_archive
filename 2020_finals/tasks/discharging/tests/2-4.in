1407
144142 310245 532160 1356620 2915113 3281598 3838313 4742723 5251452 5893063 6169736 9195588 9522080 9739856 10993812 11056746 11268027 11441437 11950892 13138144 13738451 13812780 14499145 15441339 17044741 17544326 18038434 18465875 18517536 19210374 19551644 20240009 20299456 20808953 22006036 22523595 22657352 24513613 24639171 25027873 25321703 25584719 28187762 28473317 29341254 29513515 29677806 30080329 30326343 30407388 30701308 31084343 32231609 32660709 32935469 33499058 33732246 34005975 34136248 35164677 35344121 35349189 35840472 36222664 37806667 37949162 38610293 39043668 39615335 41789243 41931253 41961345 42221119 42646628 43383064 43602158 43834856 44575560 44696842 44710524 44861200 45665883 47727368 48242163 48879476 49858694 50095530 50952485 52041581 54361881 54547254 55178279 55432046 55463212 56513366 56986640 57305910 57323771 59358241 59421343 59813432 59988547 60141030 60613605 61149554 61933059 62637420 62865130 63492172 64022400 64089101 64808913 65166627 65328987 65982286 69407705 70718627 71915860 72534162 72894749 73489444 76059120 77730361 78270064 78979651 79096449 79439128 80317165 80786623 82052869 82512253 82701671 83541363 83882634 84157104 84353065 85465892 85874528 86771869 87393772 87717444 88931194 90402658 90456765 90646456 91742325 91834914 94661734 94736340 96372162 96674316 97080047 97113916 97279789 97834528 97857991 98229102 98854426 99883890 99891787 103189706 103332969 103367724 103571824 103716691 104958741 105924588 106863781 107383447 107512804 107930750 108128363 108789434 109034231 110147064 112603686 112761520 112992841 113969427 114332752 114851342 115914370 116224619 116674619 116762219 116858093 117095558 117472152 117674006 118778555 118815631 119833578 120726120 120801733 120973005 121139787 121652067 122127429 122445585 123033530 123036199 124864405 127046095 128000194 129098644 130022224 130195626 130327440 131365655 131660323 132001470 132085344 132140654 132284715 132404562 133331270 133879360 134142044 134480405 134893000 136046593 138119313 138216885 138482908 140169116 141878823 142742579 143575531 144386333 144669690 146764561 147916096 148667815 152221437 152605110 153389927 155134080 155138511 155667250 156263040 156481503 156506705 157349027 158624423 159564103 160268072 160711999 160903811 161636881 161660782 161959624 162076320 162965730 163132202 164036423 164750438 165437626 166744362 168039191 168283973 168386573 168970302 171007487 172080299 172925515 174230495 175474405 175689001 176959748 177584792 178752582 180500320 181380659 181760896 182867462 182977281 183333524 183666096 184127596 185773177 186507101 188139539 189193835 190445623 190626212 190837927 191409428 193065190 193146318 194960975 195644672 196439168 197193503 197797022 198238983 200390059 200883470 202011632 202492277 203664697 204396725 205379914 207476536 207883914 208765443 208887321 208939814 209288990 210763898 211318745 211576335 212011457 213202405 213706104 214014577 214052969 214713187 214902984 214965733 215866953 216614538 216906472 216917815 217731036 217825756 217947730 217957787 218451143 219238653 220048479 220406722 220538958 221066412 222385319 222446153 223970437 225223400 225563407 226148234 226297415 226316395 227939885 228866414 230028635 230149681 230994321 233214256 233537261 234890181 236031204 237431696 239053698 240957700 241264164 241328178 241528907 242429391 243768910 243912187 246779797 247031416 247065764 247458908 247654240 248034564 251422422 251913805 252608992 253970342 254414663 255669128 257131095 257482536 258060494 259170585 260801517 261520537 263859168 264242345 264743886 265559030 267900101 268459433 268532119 268551176 268917536 269677047 270253338 270853930 270935910 271246255 271692625 273165646 273806253 276120366 276272895 276525380 276623434 276769382 280931034 282235234 283646569 285954949 287242092 288880397 288988492 289538427 290316772 290858774 291486356 291904056 292452477 292607110 292791382 293067013 293275256 293812894 293962554 295754202 295998956 296920032 297096829 297129280 297845483 298048127 298059108 298075068 298928259 299248608 299787149 300055692 300069627 300264203 300882442 301037949 301210657 301513549 301840784 302058624 302490628 302935562 303121784 303512266 304025766 304522864 304622777 305289764 306003780 306485581 307545128 308581113 309195537 312839709 315038883 315045328 315312274 315444260 315697269 316061591 316170346 317008768 320062090 322337813 324640311 325491708 325794092 326042234 326929126 327075345 327123957 327409206 328203961 328674336 330046480 330700857 331177203 333669324 335528379 336061686 336152756 336500852 337729003 337812808 337958827 338567597 338871596 340654985 341240688 341553031 341607553 341853886 342409364 342633435 342646768 342891750 345747913 346185287 346384559 346523790 347058957 347102577 347516507 347612881 347981486 347999620 348305101 349203529 349390142 350160323 351264493 351319567 352469124 353316110 353991747 354075195 355433678 355734229 356293685 356728166 357875722 358006718 358171305 359403641 359549951 359575613 361511548 362302132 363321159 364078771 364999149 365695964 365823687 365951105 365993537 366355478 366718958 368128101 368611096 369446230 369898138 371088616 371549178 371929623 372248526 372453156 375994895 376556552 376619276 380440881 380513246 381317852 382072437 383745949 384186719 384287941 384425845 384730642 384916693 385069017 385094404 385454637 385970651 386843376 387480330 388463666 389143819 389331438 389794306 389955565 391527319 392934078 393196083 393240134 394359125 394604931 394758530 394873063 395367633 395550640 395761164 396697008 396718092 396787909 396846442 397810574 398632193 398933564 399525549 399765536 402844509 403100005 403196081 404787671 404872057 405873609 406222900 406371579 408509888 408979668 409065650 409079110 410036327 412078256 412167740 412437548 413262293 413412447 414642881 417425560 419429449 419445561 419458346 420255039 421567336 422455159 422475105 423265144 423566147 424428742 425614367 426242019 426324728 426541986 426591738 427968321 430849506 431258687 431625086 431697500 432881719 435202720 435331244 438611384 441508525 443480914 444261474 445287104 445615622 447179229 448138374 448214289 448972709 450806754 451017873 451182573 451415092 452351211 452481209 452599651 452990165 455631120 455873040 456365299 456689204 458589142 458720491 459696812 459714335 459725920 459999957 460381241 461712515 463480849 463653746 463676418 463828360 464160995 464186694 464778890 465532913 465617628 467104523 467315066 468415551 468590632 468641533 470765845 471963932 473786498 476179696 477860254 481351988 481626976 482029204 482052883 482063983 482130582 482431319 484232326 484249955 487156312 488302799 488399215 488422444 488428626 489509393 490553662 491063121 491109503 491776160 492288263 492522969 492582496 493357279 494547217 496820601 497377620 498848379 499193544 500593944 500771052 500877002 501700501 502328437 502404259 503375109 505199368 505681252 505795598 505801457 507736337 507940355 509259859 511282098 512312964 512770759 515139348 515456169 518113750 519169088 519531113 519543152 520055113 521004619 521421961 521891173 522124040 522639193 523725927 523890781 523999272 524682119 524784474 524810038 525047482 525336423 525449512 525707728 525917644 526898248 526914487 526926132 530190509 531523862 532131017 532306673 532728553 533553311 535070362 535580140 536980611 537111297 537449412 539536645 539774536 540484680 541361890 541899840 542564249 542984892 543348496 543606767 543948257 545782941 545859952 545868401 547380028 548156397 548812089 549511532 550062694 552586524 553009420 553457367 554267127 555675065 556432235 557785565 558509588 558810024 558999016 559088414 559546351 561399767 561980282 564452529 565591663 567808044 567972517 568214526 568700858 571172493 571228007 572121555 572719046 572770958 572795637 573553674 574728785 575015202 577265348 579416057 580137149 581192701 581667685 581927355 582270495 582910070 584824011 585003403 585025034 585174202 586293790 586950184 587497676 587586306 588388228 588776659 590283405 591081297 591285154 591870392 592602943 595192748 595414809 596402573 602730327 602933518 603410853 604180972 604345082 604920141 604987123 605559616 605670167 608239892 608486655 609568294 609832401 610145214 610683626 612434191 613203980 614715383 615895451 615924919 615983918 616026228 616859456 617082232 617326241 621168411 621780134 623122047 623850940 624835369 625161872 625612370 625682816 626115006 626653497 627802439 628016309 629284505 629397968 630174363 630396992 631158068 631339137 632829291 633793548 633797588 634501300 634722323 635145243 635529044 635695682 636452104 636684177 636850390 637437058 638801328 638856458 639897979 640623605 641303935 641913081 642121006 642313556 643223943 643774351 643930145 644471559 645084590 645405043 646099146 649694669 650117628 650751738 651171806 652180980 652620448 653032480 653187836 653878938 655333937 657178010 657266580 657312272 658396058 659356207 659487726 659848955 659899184 661219480 662108584 662388957 662937982 663380680 664814374 664934115 666698829 667370434 668676674 669287687 669956926 670143379 670197733 670550039 671643338 672081391 672315030 672689221 672838514 673760285 674772289 674846699 676226385 676604209 676750624 676896139 677474555 677695886 678128036 678276490 679921919 680008322 681667694 682139836 682426779 683320240 683382035 683479304 684274126 684610208 685647713 686383356 687145726 689968445 690025122 691159314 691175386 691561759 691687202 691741765 692658584 693854965 694280612 698132677 698411378 698817691 699872277 699904250 701078161 702231551 702583272 703758987 704097838 705179629 705821229 706703737 706724585 707818713 707850762 708842097 708862677 710415039 711741662 711909897 712201368 714016740 714349131 714637547 714651020 714785119 715554153 717322806 717985550 717988244 718380371 719115929 719853828 720141886 721385863 721617767 721720508 722074913 723977833 724173760 725228405 726277883 727018881 727224490 727822088 727843640 728503627 728649350 728780007 730433883 731255250 732507945 733129433 733800464 733945885 734118112 734332833 734591644 734678468 734925384 734973469 735359380 735489909 736620627 738125532 738398393 740494414 740655164 741421437 741855256 741984399 743078477 743323830 743587032 743836279 744381021 745020514 745064665 745128515 745949034 746534357 746567893 747426199 747475758 748109435 748594023 749653955 750504131 750638223 750791942 751853910 752521976 753798975 754161409 754636890 754691532 754986923 757136696 758242610 758786941 759472740 759817413 760095514 761283396 761831835 762336683 762565991 763195507 764067385 765230153 766919345 767496713 767556767 767756267 767995388 768666060 769961783 770300529 770310815 770583111 770713453 772334360 773037136 776985899 777511792 778491253 779227069 779270337 780586915 781188717 781547711 781736803 782351082 782376318 783420000 784743000 785882752 786562871 786936257 787396279 787704633 787784234 789097813 789756365 789841243 791405752 792242091 793996849 794346892 795177228 796006662 797458197 797596560 797981509 798030955 798227243 799794754 800599425 801046983 801697442 803041722 803129363 803257989 803320290 803388871 804795908 805044038 805252258 806356136 806544279 806970112 807447269 808541934 808550576 808576161 808649959 809535314 810868694 811578359 812093093 812602314 812814825 813189892 814081612 815130116 815423932 815970842 816744971 816882716 818063952 818403934 819123340 820305811 820521377 820570880 822060757 822230814 822438053 822718864 825050469 826475047 827087466 827672177 827986186 828082538 828979608 829045079 830748186 831068961 831184063 831637463 832217632 832371378 832417326 832736306 832783942 833496253 835986851 836357137 838381334 838502924 838617364 838858872 839607305 839987283 840596924 840797687 841125904 841459581 841481972 842862740 843355118 843733051 844846289 845151768 845899452 846055711 847665070 847955826 848140642 848664805 849196869 851232965 851363459 852136627 852233709 853821138 854117410 854171872 854652149 854925508 857775194 858063568 858290850 858507444 859060990 859422857 859599101 861397752 861650518 862358401 862622288 864704657 866329458 869451921 870721122 871058192 872454599 873745375 875408988 875474711 875991342 876218352 877297757 880713011 881078790 881179719 881207637 882354129 883610645 884087106 886210349 886656071 887423925 888606876 888974246 891293318 891643882 891913730 891954516 892841485 892903460 893807247 893965343 897408798 897939735 898102980 898477141 899260380 899277549 899777313 899962333 900112926 901582148 902570911 903451815 904251358 904625628 906303315 906804001 907978710 908064709 908149364 909632448 910011910 910235252 910772488 913252756 915371705 915748773 916264126 918315862 919181532 919561331 922265811 922552013 924061511 924095308 924252265 924595600 925933679 927193043 927652195 928821482 929426863 930361088 930483601 930775753 930847100 931298246 931433273 932022380 932962711 933350242 934283644 934975352 937337826 937491879 938339310 938743151 939527206 939761138 940140470 940388828 941191645 942196310 942457244 943042701 943663183 944143641 944657907 945602264 945755709 947332688 947397357 949754401 951393232 953679857 954869433 955141787 955214634 955734681 956569657 956650333 957663295 957963611 958294354 958839060 960355826 960582787 961279018 961497797 962315610 962921631 963718317 963843744 964307369 964737600 964807555 967489803 967687695 968280323 969888903 970568652 971374395 972157530 974175385 974607642 975719216 975814690 975860709 975895982 977559475 979061587 979770385 981037419 981053861 981719062 983753387 984284610 984311943 984935942 985148129 985723106 986025930 986272701 986968737 987698514 990579285 992283022 993209913 993404607 994104558 996565924 996599900 996836273 997501375 997807248
