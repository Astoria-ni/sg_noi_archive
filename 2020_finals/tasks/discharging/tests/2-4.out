1372170659598
